### DISCUSSION TITLE
Where to put Generic.xaml

### DISCUSSION THREAD

**Discussion by hxhgxy** [Excel-DNA/ExcelDna#638] [Q&A]
In my NET6.0 class library project, where can i put Generic.xaml?

if i put it or any xaml for controls in Window or UserControl.Resources, there would be error.

Thanks,

**Comment by hxhgxy** [comment]
![net6 issue2](https://github.com/Excel-DNA/ExcelDna/assets/37101628/8fae65cf-5622-43d5-9f7d-f7801ff4f739)
I created a WPF Application, Net 6.0. In the project i put all stuff of excel dna for ribbon. It runs normally if I run it with the project normally.

![net6 issue1](https://github.com/Excel-DNA/ExcelDna/assets/37101628/ee3d8603-2812-44e2-bf37-54ec81d6173f)
but, if i run it with Excel, error come out: cannot find the style file.

![net6 issue3](https://github.com/Excel-DNA/ExcelDna/assets/37101628/cbe44d98-7e50-4495-af8e-902d5d88c374)
This is the Excel Ribbon. Once I click the button to load WPF window, I will get the error.

**Comment by hxhgxy** [comment]
this is not an issue in project NET Framework 4.8.

**Comment by govert** [comment]
This is quite far beyond what I know about WPF and how it loads under .NET 6.

**Comment by hxhgxy** [comment]
Thanks anyway!
this is not problem with ExcelDNA, but the Net6.0 class library. 
Cannot refer to UI dll, for example, materialdesign UI downloaded via nuget.
but, can refer to any resource dictionary file in the class library project itself.
